### PR TITLE
Add policy for PR reviews

### PR DESCRIPTION
**Pull request by adborden**
This came out from our retro today. Leaving this in draft status for folks to review and comment.

### CODE REVIEW

**Comment by adborden** [comment]
@jbrown-xentity you missed the discussion around this. Not sure if it captures everything you were wanting to capture.

**Review by jbrown-xentity — COMMENTED** [review]
Want to note a number of exceptions:
1. There are a number of instances where CI is failing, but have been told it is ok to approve request
2. If the reviewer is not an expert on the purpose or language of the changes, how can they verify the code is in a deployable state? Should we approve if simple edits/fixes and seem like everything should work?

**Comment by adborden** [comment]
Good points for clarification...

For CI, how about we drop the part about "passing tests in CI" since GitHub is enforcing that CI checks are passing. So adding something like:

> Any critical CI checks should be enforced by GitHub on protected branches (`develop` and `master`), so it's not required that CI checks are passing in order to approve a PR. Instead, it's important that tests have been added and they are running in CI.

On expertise:

> Data.gov encompasses many technologies (too many, in fact) and it's not practical to have everyone be an expert at everything nor to have only a single expert review all code in a specific domain e.g. PHP. Any developer should be able to follow the README in order to get a working system and should post on the PR if that doesn't seem to be the case.
>
> Ideally, our tests should give us good confidence that changes are working correctly. Even though that is currently not the case (we are working on building up our test coverage), it's not required to try out the code locally.

And finally, I want to note that this is why getting Jenkins setup is so critical. With continuous delivery to the sandbox on pushes to `develop`, that should give us good confidence as to whether or not `develop` is in a deployable state.

**Comment by jbrown-xentity** [comment]
Absolutely, love those additions. And +1 to Jenkins on sandbox!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -214,3 +214,22 @@ Branch | Deployed to | Frequency
 `develop` | AWS sandboxes | manual
 `release/*` | BSP dev | manual
 `master` | BSP prod | manual
+
+
+## Pull requests
+
+Developers should feel empowered to review each others code, even if you're not
+an expert on a particular application or feature. Any developers on the team can
```
Comment (mogul) [inline CONTRIBUTING.md]: s/developers/developer/

**Review by mogul — COMMENTED** [review]
Looking forward to the additions from the conversation above, as that's extra-clear about what we're after!

**Comment by adborden** [comment]
Okay, I've rebased and incorporated the changes discussed. This is ready for a final :+1:

**Review by mogul — APPROVED** [review]

**Pull request merged**